dm/syncer/dbconn/utils.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dbconn
+
+import (
+	"fmt"
+
+	tcontext "github.com/pingcap/tiflow/dm/pkg/context"
+	"github.com/pingcap/tiflow/dm/pkg/terror"
+)
+
+// GetTableCreateSQL gets table create sql by 'show create table schema.table'.
+func GetTableCreateSQL(tctx *tcontext.Context, conn *DBConn, tableID string) (sql string, err error) {
```

Comment:
we already have this func in `dm/pkg/utils/db.go`

Reply:
That function use base conn, we would better use a wrapped conn. but there may has `import cycle not allowed` when we use a wrapped conn in package utils. so i create this utils in syncer/dbconn, maybe we can add more functions which need use a wrapped conn in here.

Reply:
i don't think handel-err cmd needs auto retry when sth wrong with db connection  and there are so many `utils.go` files which contains operations for db  in our project now, add a new one again might not be a good choice 🤣

 btw, if you really want to add retry for this, maybe you can add a func like https://github.com/pingcap/tiflow/blob/b581f7811880777df3c850d7f19efae3232ee86a/dm/pkg/conn/utils.go#L26
 
wait other reviewers suggestions

Reply:
Looks no big problem to me.